lib/commons/color/get-background-color.js

```diff
@@ -44,16 +45,27 @@ function getBgColor(elm, elmStyle) {
 	return bgColor;
 }
 
-function calculateObscuringAlpha(elmIndex, elmStack) {
+function contentOverlapped(targetElement) {
+	// Get dimensions of text content box.
+	// Subsequent clientRects are bounding boxes, if different than content box
+	var targetRect = targetElement.getClientRects()[0];
+	var elementAtPoint = document.elementFromPoint(targetRect.left, targetRect.top);
+	if (elementAtPoint !== null && elementAtPoint.innerHTML !== targetElement.innerHTML) {
```

Comment:
The strict comparison was failing on `elementAtPoint` === `targetElement` so I changed it to   a loose equality check and jshint was complaining about it. So I changed it to innerHTML for both of those reasons.